tracer/test/snapshots/DataStreamsMonitoringRabbitMQTests.HandleProduceAndConsume.verified.txt

```diff
@@ -1,96 +1,340 @@
 ﻿[
+  {
+    EdgeTags: [
+      direction:in,
+      topic:TopicQueue2,
+      type:rabbitmq
+    ],
+    Hash: 304252030283527403,
```

Comment:
It's quite hard to visualize these, Any chance you could add a quick diagram (ascii or anything) to show the messages flow through in the sample? That would make it easier to verify these are correct 🙂

Reply:
I added a comment about the actual pipelines that will be generated by these, and the expected points / tags we should see. lmk what you think!